src/device.coffee

```diff
@@ -44,10 +44,19 @@ exports.getID = do ->
 exports.reboot = ->
 	request.postAsync(config.gosuperAddress + '/v1/reboot')
 
+exec = require('child_process')
+
+findMountPointByPartLabel = (label) ->
+  command = "grep $(blkid -L #{label}) /proc/mounts"
+  try
+    return (exec.execSync(command).toString().split " ")[1]
+  catch err
+    throw new Error ("Could not determine mount point for partition #{label}: #{stderr}, ExitCode: #{error.code}", { error: body.Error }, 'Set log to display error')
```

Comment:
The `Error` constructor only takes one argument, I'm not sure what the second two actually are either?